content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "New design, UX and extensibility digest for login page et. al."
+tags:
+- core
+- developer
+author: scherler
+---
+
+== Target audience
+
+The first part of the blog post is an introduction to the new design and UX to the general audience.
+The later part is talking about extensibility in a more technical manner.
+
+
+== Overview
+
+The recent changes to some core pages provide new design and UX and further dropping all external dependencies to prevent
+any possible malicious javascript introduced by third party libraries. 
+This never had been an issue with current installation of jenkins but having read https://hackernoon.com/im-harvesting-credit-card-numbers-and-passwords-from-your-site-here-s-how-9a8cb347c5b5[Why?] the author of the changes believes that the article has good points and leading by example may raise awarness of data protection. 
+
+This meant to drop the usage of the jelly layout lib (aka `xmlns:l="/lib/layout"`) and as well the page decorators it
+supported. However there is a new SimplePageDecorator which can be used to modify the look and feel for the login and sign up page.
+
+The following pages have given a new design:
+
+- HudsonIsLoading / HudsonIsRestarting
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_a.png[role="center"]
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_1_b.png[role="center"]
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_2.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_3.png[role="center"]
+
+
+== UX enhancement
+
+Form validation has changed to give inline feedback about data validation errors in the same form.
+
+- Login
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_4.png[role="center"]
+
+- Sign up
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_5.png[role="center"]
+
+The above image shows that the validation is now done on all input fields instead of before breaking on the
+first error found, which should lead to less retry cycles.
+
+Instead of forcing the user to repeat the password, the new UX introduces the possibility to display the password in
+clear text. Further a basic password strength meter indicates the user the password strength while she is writing it.
+
+image:/images/post-images/2018-06-19-JENKINS-50447/JENKINS-50447_6.png[role="center"]
+
+== SimplePageDecorator
```

Comment:
A better headline would be "customizing the UI" (or similar). Again, audience confusion.